app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/indexActionsBlock/indexActionsBlock.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { showModalAction } from 'controllers/modal';
+import { appInfoSelector } from 'controllers/appInfo/selectors';
+import classNames from 'classnames/bind';
+import { GhostButton } from 'components/buttons/ghostButton';
+import { projectAnalyzerConfigSelector } from 'controllers/project';
+import styles from './indexActionsBlock.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  indexActionsBlockTitle: {
+    id: 'IndexActionsBlock.title',
+    defaultMessage: 'Actions with index',
+  },
+  removeIndexDescription: {
+    id: 'IndexActionsBlock.removeIndexDescription',
+    defaultMessage:
+      'All data with your investigations will be deleted from the ElasticSearch. For creating a new one you could start to investigate test results manually or generate data based on previous results on the project once again',
+  },
+  generateIndexDescription: {
+    id: 'IndexActionsBlock.generateIndexDescription',
+    defaultMessage:
+      'All data is removed from ElasticSearch and new one is generated based on all previous investigations on the project in accordance with current analysis settings. You can start auto-analyse test results after receiving an e-mail about the end of the generation process',
+  },
+  removeIndexButtonCaption: {
+    id: 'IndexActionsBlock.removeIndexButtonCaption',
+    defaultMessage: 'Remove index',
+  },
+  generateIndexButtonCaption: {
+    id: 'IndexActionsBlock.generateIndexButtonCaption',
+    defaultMessage: 'Generate index',
+  },
+  generateIndexButtonProgressCaption: {
+    id: 'IndexActionsBlock.generateIndexButtonProgressCaption',
+    defaultMessage: 'In progress...',
+  },
+  analyzerDisabledButtonTitle: {
+    id: 'IndexActionsBlock.analyzerDisabledButtonTitle',
+    defaultMessage: 'Service ANALYZER is not running',
+  },
+});
+
+@connect(
+  (state) => ({
+    indexing_running: projectAnalyzerConfigSelector(state).indexing_running,
```

Comment:
`indexingRunning` (we use camelCase)

Reply:
I discussed it with Yana before send pull request. We decided to leave "_" - it simplify to submit form, because such field name came from server. See the screenshot below. Should I change it?
![image](https://user-images.githubusercontent.com/31861805/41410179-3f115736-6fe1-11e8-9923-8ec6975bc9da.png)

Reply:
Such approach also used in component on screenshot below.
![image](https://user-images.githubusercontent.com/31861805/41410381-f97cf968-6fe1-11e8-8b38-201398749b55.png)